circuits/types.go

```diff
@@ -14,3 +14,7 @@ type BallotMode[T any] struct {
 	CostFromWeight   T
 	EncryptionPubKey [2]T
 }
+
+func (bm BallotMode[T]) Bytes() []byte {
```

Comment:
what is this?

Reply:
i needed this mock until i implement BallotMode properly in statetransition circuit. i'll do in a future PR